pygdf/datetime.py

```diff
@@ -0,0 +1,91 @@
+import numpy as np
+from libgdf_cffi import ffi, libgdf
+import weakref
+from . import _gdf, columnops, utils, cudautils
+from .buffer import Buffer
+from .cudautils import compact_mask_bytes
+
+
+class DatetimeColumn(columnops.TypedColumnBase):
+    def __init__(self, data, mask=None, null_count=None, dtype=None):
+        # currently libgdf datetime kernels fail if mask is null
+        if mask is None:
+            mask = np.empty(data.mem.size, dtype=np.bool)
+            mask[:] = True
```

Comment:
Couldn't the above two lines be combined to be `mask = np.ones(data.mem.size, dtype=bool)`